packages/components/src/toolbar/style.scss

```diff
@@ -66,8 +66,13 @@
 			}
 		}
 
-		// Focus style.
-		&:focus::before {
+		// Focus style but toolbar buttons, execpt block movers.
+		&:not(.block-editor-block-mover-button):focus::before {
+			@include block-toolbar-button-style__focus();
+		}
+
+		// Focus style for block mover buttons (using :focus-visible instead of :focus)
+		&.block-editor-block-mover-button:focus-visible::before {
```

Comment:
I attempted to move styles to `packages/block-editor/src/components/block-mover/style.scss` as you suggested:

- I still had to leave those same styles in place in `packages/components/src/toolbar/style.scss` because they are applied to the rest of toolbar button (using `:focus`)
- I had to add an override in the block mover button styles to remove the `:focus` styles applied by the block toolbar
- I had to leave some minor overrides for the size/position of the `::before` pseudo element in the list view

WDYT ?

Reply:
Whoa, talk about leaving things cleaner than you found them 👏 Great refactor!